app/helpers/layout_helper.rb

```diff
@@ -505,13 +505,17 @@ def new_child_fields_template(form_builder, association, options = { })
 
     content_tag(:div, :class => "#{association}_fields_template form_template", :style => "display: none;") do
       form_builder.fields_for(association, options[:object], :child_index => "new_#{association}") do |f|
-        render(:partial => options[:partial],
-               :layout => options[:layout],
+        render(:partial => options[:partial], :layout => options[:layout],
                :locals => { options[:form_builder_local] => f }.merge(options[:form_builder_attrs]))
       end
     end
   end
 
+  def card_status_mini(options)
+    options.reverse_merge! :count => '', :icon => '', :callback => '', :size => 'col-md-4', :notifications => ''
+    render :partial => 'common/card_status_mini', :locals => options
```

Comment:
They change the dropdown now. 
Not sure about size, I could make them smaller but we would have extra white space on the side.
I tried adding the background and I think it looks really bad to have a grey area in the middle of the white page.